src/scaffoldmaker/meshtypes/meshtype_3d_stellate1.py

```diff
@@ -0,0 +1,543 @@
+"""
+Generates a 3-D stellate mesh with cross arms radiating from a central node, and variable numbers of elements along each arm.
+"""
+
+from __future__ import division
+import math
+from opencmiss.utils.zinc.field import findOrCreateFieldCoordinates
+from opencmiss.zinc.element import Element
+from opencmiss.zinc.field import Field
+from opencmiss.zinc.node import Node
+from scaffoldmaker.meshtypes.scaffold_base import Scaffold_base
+from scaffoldmaker.utils.eftfactory_bicubichermitelinear import eftfactory_bicubichermitelinear
+from scaffoldmaker.utils.meshrefinement import MeshRefinement
+from scaffoldmaker.utils.eft_utils import remapEftNodeValueLabel, scaleEftNodeValueLabels, setEftScaleFactorIds, remapEftLocalNodes
+from scaffoldmaker.utils.matrix import rotateAboutZAxis
+from scaffoldmaker.utils.vector import magnitude
+
+class MeshType_3d_stellate1(Scaffold_base):
+    """
+    classdocs
+    """
+    @staticmethod
+    def getName():
+        return '3D Stellate 1'
+
+    @staticmethod
+    def getDefaultOptions(parameterSetName='Default'):
+        return {
+            'Number of arms' : 3,
+            'Number of elements in long arm' : 4,
+            'Number of elements in short arms' : 2,
+            'Element length along arm' : 1.0,
+            'Element width across arm' : 0.5,
+            'Element thickness' : 0.5,
+            'Extent of indentation around centre' : 0.5,
```

Comment:
This is not easy to understand. Why not have a scale factor on the dip between arms, with 1.0 the default which makes the dip 0.5*(elementLength + elementWidth) away -- see code below which does this.